Specs/DateTime/Dutch/DateExtractor.json

```diff
@@ -1467,15 +1435,14 @@
     ]
   },
   {
-    "Input": "Ik ga 21-04-'16 terug",
-    "NotSupported": "dotnet",
+    "Input": "Ik ga 21/04/16 terug",
```

Comment:
Is 21-04-16 covered somewhere else? 14-04-2011 and 14.04.2011 are very common.

https://taaladvies.net/taal/advies/vraag/1489/datumnotatie/

Reply:
I think this has been changed to match the English case "i'll go back 21/04/16". There are other test cases covering the patterns with "-" and "." separators.